integrationExamples/gpt/amp/README.md

```diff
@@ -1,3 +1,10 @@
+##WARNING
+The below documented method of deploying prebid on AMP requires remote.html
+This is being deprecated on March 29th. A new method the requires Prebid Server
```

Comment:
Should the year be added to the date?

Reply:
Good idea, especially for long term (someone reads this in 2019). Added.